CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.12)
 
 project(icub_firmware_shared
-        VERSION 1.40.0)
+        VERSION 1.40.1)
```

Comment:
hi @ale-git, I have changed the icub_firmware_shared VERSION to be 1.40.1 so that we maintain 40 as a value of the iCub Software release and we use the final .1 to force icub-main to use this particular version.